from data.services.exceptions import UnknownError

TOKEN_WAS_NOT_ENTERED = "Войти не удалось: токен не найден. Введите токен."

INVALID_TOKEN = "Войти не удалось: неверный токен."

UNKNOWN_ERROR = "Что-то пошло не так"

LOG_IN__SUCCESSFUL = "Вход успешно выполнен"
